### PR TITLE
Starting on contributor guidelines

### PR DESCRIPTION
**Pull request by gvwilson**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
+Amy is an open source project, and we welcome contributions of all
+kinds: new lessons, fixes to existing material, bug reports, and
+reviews of proposed changes are all equally welcome.
+
+By contributing, you are agreeing that Software Carpentry may
+redistribute your work under [this license][license].  You also agree
+to abide by our [contributor code of conduct][conduct].
+
+If you would like to know what we need help with, please see the
+[issues][issues] on GitHub.
```
Comment (wking) [inline CONTRIBUTING.md]: Where the anchor name matches the link text, you can leave the anchor name blank.  So `[issues][]`.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+## Getting Started
```
Comment (wking) [inline CONTRIBUTING.md]: This seems like user-facing information to me.  Why remove it from the README?

**Comment by wking** [comment]
On Wed, May 27, 2015 at 07:09:08PM -0700, Greg Wilson wrote:
>     A CONDUCT.md (33)

This is copied from another SWC repository, no?  Maybe mention which
one in the commit message, so we know where to look for updates.

File: CONTRIBUTING.md (hunk 1 above)
Comment (gvwilson) [inline CONTRIBUTING.md]: Do we prefer `[anchor][]`, or `[anchor][anchor]` ?

**Comment by gvwilson** [comment]
The original source is contributor-covenant.org, so I thought I should 
cite that, but yeah, linking to the lesson-template is a good idea - 
I'll fix.

File: CONTRIBUTING.md (hunk 2 above)
Comment (pbanaszkiewicz) [inline CONTRIBUTING.md]: +1 — I definitely expect this kind of information to appear in README when I look up new projects.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
+Amy is an open source project, and we welcome contributions of all
+kinds: new lessons, fixes to existing material, bug reports, and
```
Comment (pbanaszkiewicz) [inline CONTRIBUTING.md]: "new lessons"

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+4.  Enjoy your new version of Amy:
+
+    ~~~
+    $ make serve
+    ~~~
+
+## Coding Guidelines
+
+*   Format all dates as YYYY-MM-DD.
+
```
Comment (pbanaszkiewicz) [inline CONTRIBUTING.md]: I'd add:

* we expect PEP8 compatibility with new code
* no PEP8-ing historical code unless the meaning of that code is being changed in this specific PR
* PEP257 encouraged
* unittests

**Comment by gvwilson** [comment]
Shall I go ahead and merge before it goes stale?

File: README.md [hunk 5]
```diff
@@ -2,56 +2,68 @@
...
-    You may need some additional dependencies to install [Bower](http://bower.io/), such as [NodeJS](https://nodejs.org/) and [npm](https://www.npmjs.com/).
+    You may need some additional dependencies to install [Bower][bower], such as [NodeJS][nodejs] and [npm][npm].
 
-    **Note**: if you don't want to use `sudo`, you can install `bower` locally. You'll need to set up your `$PATH` correctly, though. Look [here](https://docs.npmjs.com/getting-started/fixing-npm-permissions#option-2-change-npm-s-default-directory-to-another-directory) for details.
+    **Note**: if you don't want to use `sudo`, you can install `bower`
+    locally. You'll need to set up your `$PATH` correctly, though. Look
+    [here][fixing-npm-permissions] for details.
 
-4. Setup your local database wit redacted (development-ready) data with:
+4.  Setup your local database wit redacted (development-ready) data with:
```
Comment (pbanaszkiewicz) [inline README.md]: This PR (#314) is a conflict because we introduced @wking fix to this line ("wit" → "with").

**Comment by pbanaszkiewicz** [comment]
Sorry it took so long, GH doesn't send messages about updated code in PR :/

I'm fine with this PR after you resolve conflict by rebasing against master.

**Comment by gvwilson** [comment]
Done - push the button?

File: README.md [hunk 6]
```diff
@@ -2,56 +2,68 @@
...
-    You may need some additional dependencies to install [Bower](http://bower.io/), such as [NodeJS](https://nodejs.org/) and [npm](https://www.npmjs.com/).
+    You may need some additional dependencies to install [Bower][bower], such as [NodeJS][nodejs] and [npm][npm].
 
-    **Note**: if you don't want to use `sudo`, you can install `bower` locally. You'll need to set up your `$PATH` correctly, though. Look [here](https://docs.npmjs.com/getting-started/fixing-npm-permissions#option-2-change-npm-s-default-directory-to-another-directory) for details.
+    **Note**: if you don't want to use `sudo`, you can install `bower`
+    locally. You'll need to set up your `$PATH` correctly, though. Look
+    [here][fixing-npm-permissions] for details.
 
-4. Setup your local database with redacted (development-ready) data with:
+4.  Set up your local database wit redacted (development-ready) data with:
```
Comment (pbanaszkiewicz) [inline README.md]: Still seeing "wit" here. I'm guessing you did some sort of a merge instead of rebase.

**Comment by pbanaszkiewicz** [comment]
:+1:

**Pull request merged**